azure-pipelines.yml

```diff
@@ -3,31 +3,43 @@
 jobs:
 - job: Test
   pool:
-    vmImage: 'ubuntu-18.04'
+    vmImage: 'ubuntu-latest'
   steps:
   - task: UsePythonVersion@0
     inputs:
       versionSpec: "3.7"
       architecture: x64
+  - script: echo "##vso[task.prependpath]$CONDA/bin"
+    displayName: Add conda to PATH
+  - script: conda create --yes --quiet --name physlr
+    displayName: Create Anaconda environment
   - script: |
-      curl https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
-      sudo apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-7 main"
-      sudo apt-get update
-      sudo apt-get install -y --no-install-recommends clang-format-7 clang-tidy-7      
-      sudo ln -sf clang-format-7 /usr/bin/clang-format
-      sudo ln -sf clang-tidy-7 /usr/bin/clang-tidy
-    displayName: Install clang-format and clang-tidy
+      source activate physlr
+      conda install --yes -c conda-forge -c bioconda --file requirements.txt
+    displayName: Install dependencies
+  - script: |
+      source activate physlr
+      conda install --yes -c bioconda -c conda-forge clang clang-tools openmp
+    displayName: Conda Install clang-format and clang-tidy
   - script: sudo apt-get install -qq libboost-dev
     displayName: Install boost
   - script: pip3 install pylint .
     displayName: Install Python packages
   - script: make lint
     displayName: Run pylint
-  - script: make -C src all
+  - script: |
+      source activate physlr
+      export CPPFLAGS="-isystem $CONDA_PREFIX/include/ $CPPFLAGS"
+      export LDFLAGS="-L$CONDA_PREFIX//lib -lbtllib $LDFLAGS"
```

Comment:
Are these needed? I believe the flags should be set when you activate the environment?